cli/azd/pkg/config/config.go

```diff
@@ -31,3 +33,98 @@ func GetUserConfigDir() (string, error) {
 
 	return configDirPath, err
 }
+
+type Config struct {
```

Comment:
Would be good to get a quick team consensus on a single user-level config, and see if this makes sense for future config storage.

Reply:
Agreed - created this as a placeholder for the uber user-level config with the values I need for now.  We can easily add additional values in here as needed.